buidlerCoverageSol4.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
 
   paths: {
     sources: "./contracts",
-    artifacts: ".coverageV4Artifacts"
+    artifacts: ".coverageV4Artifacts",
```

Comment:
not sure why use coverage file?

Reply:
should not change this file. will remove